ledger/participant-state/kvutils/tools/codec-benchmark/src/benchmark/scala/com/daml/lf/benchmark/BenchmarkWithLedgerExport.scala

```diff
@@ -46,7 +47,9 @@ abstract class BenchmarkWithLedgerExport {
           }
         case Envelope.SubmissionMessage(submission)
             if submission.getPayloadCase == DamlSubmission.PayloadCase.TRANSACTION_ENTRY =>
-          builder += submission.getTransactionEntry.getTransaction
+          builder += TransactionOuterClass.Transaction.parseFrom(
+            submission.getTransactionEntry.getRawTransaction
+          )
```

Comment:
We problably should use `com.daml.ledger.participant.state.kvutils.Conversions` here instead directly interfacing to `com.daml.lf.transaction.TransactionOuterClass`

Reply:
Yes, I was looking at this and considered it out of scope for a couple of reasons:
1. All of this benchmark stuff sits under the `com.daml.lf` package, so either I would have to move all of it to `com.daml.ledger.participant.state.kvutils` (extends the scope of the PR), or change `private[state]` in `Conversions` (which I don't want).
2. Even then some of the stuff would use `Conversions`, while the rest would use Daml-LF (at least now).
Also, I decided to inline the `Conversions.parseTransaction`, so I can no longer use it here.

I think it would require more work to clean this up. I can create a story for that if you're fine with that. cc @remyhaemmerle-da, @fabiotudone-da, @SamirTalwar-DA

Reply:
I did not see it was a benchmark. 
Totally fine for me to let it like that.